### ISSUE TITLE
roberta fine-tuning parameters

### ISSUE THREAD

**Issue by yuyi5187** [pytorch/fairseq#3551]
I used this command to fine-tune roberta-base with MNLI train set
but I can't get the same accuracy in the paper (my accuracy was 65%)
I don't know why.  maybe this command is for trainning roberta-large
how can I make the command for roberta-base with MNLI

TOTAL_NUM_UPDATES=123873  
WARMUP_UPDATES=7432      # 6 percent of the number of updates
LR=1e-5                
NUM_CLASSES=3
MAX_SENTENCES=32       # Batch size.
ROBERTA_PATH=/path/to/roberta.base/model.pt

CUDA_VISIBLE_DEVICES=0 fairseq-train MNLI-bin/ \
    --restore-file $ROBERTA_PATH \
    --max-positions 512 \
    --batch-size $MAX_SENTENCES \
    --max-tokens 4400 \
    --task sentence_prediction \
    --reset-optimizer --reset-dataloader --reset-meters \
    --required-batch-size-multiple 1 \
    --init-token 0 --separator-token 2 \
    --arch roberta_base \
    --criterion sentence_prediction \
    --num-classes $NUM_CLASSES \
    --dropout 0.1 --attention-dropout 0.1 \
    --weight-decay 0.1 --optimizer adam --adam-betas "(0.9, 0.98)" --adam-eps 1e-06 \
    --clip-norm 0.0 \
    --lr-scheduler polynomial_decay --lr $LR --total-num-update $TOTAL_NUM_UPDATES --warmup-updates $WARMUP_UPDATES \
    --fp16 --fp16-init-scale 4 --threshold-loss-scale 1 --fp16-scale-window 128 \
    --max-epoch 10 \
    --find-unused-parameters \
    --best-checkpoint-metric accuracy --maximize-best-checkpoint-metric;
